### ISSUE TITLE
TAXSIM EITC Amount Changed for Previous Years 

### ISSUE THREAD

**Issue by GoFroggyRun** [open-source-economics/Tax-Calculator#611]
While preparing some files, I got a chance to re-execute `a2014.in` sample through TAXIMS. And I'm surprised to see that there should be SOME difference in EITC amount, even for year 2014.

Based on recent PRs, I didn't see any modification of EITC-related parameters prior to year 2015 for tax-calculator. I'm not quite sure why this is happening. But my intuition is that fixing this might be helpful to resolve the known discrepancy for EITC in 2015 (#572). 

Attached is difference file (most recent TAXSIM release) for sample `a2014.in`:
```
TAXDIFF:ovar,#diffs,#1cdiffs,maxdiff[id]=  7    127      0    -19.13 [43090]
      #big_vardiffs_with_big_inctax_diff=              126
TAXDIFF:ovar,#diffs,#1cdiffs,maxdiff[id]=  9      7      0      0.90 [19502]
TAXDIFF:ovar,#diffs,#1cdiffs,maxdiff[id]= 19    317    317     -0.01 [469]
TAXDIFF:ovar,#diffs,#1cdiffs,maxdiff[id]= 25    152     20    496.00 [22605]
      #big_vardiffs_with_big_inctax_diff=              132
TAXDIFF:ovar,#diffs,#1cdiffs,maxdiff[id]= 26      1      0   -700.00 [36539]
TAXDIFF:ovar,#diffs,#1cdiffs,maxdiff[id]= 27    157    157     -0.01 [4643]
TAXDIFF:ovar,#diffs,#1cdiffs,maxdiff[id]= 28    224    224     -0.01 [469]
TAXDIFF:ovar,#diffs,#1cdiffs,maxdiff[id]=  4    365    233   -496.00 [22605]
                       #big_inctax_diffs=              132
```

@feenberg @martinholmer @MattHJensen

**Comment by feenberg** [comment]
I updated the EITC parameters for 2015 about 12:30 pm today. If you notice 
anything you feel is incorrect, send me the 22 input variables and what 
you the value should be.

dan

**Comment by martinholmer** [comment]
Issue #611 has been resolved by the merging of pull requests #605 and #613 and #614 into the master branch.
